Signal/src/textsecure/Messages/TSMessagesManager.m

```diff
@@ -233,7 +233,7 @@ -(void)handleSendToMyself:(TSOutgoingMessage*)outgoingMessage {
     [self.dbConnection readWriteWithBlock:^(YapDatabaseReadWriteTransaction *transaction) {
         TSContactThread *cThread = [TSContactThread getOrCreateThreadWithContactId:[SignalKeyingStorage.localNumber toE164] transaction:transaction];
         [cThread saveWithTransaction:transaction];
-        TSIncomingMessage *incomingMessage = [[TSIncomingMessage alloc] initWithTimestamp:(outgoingMessage.timeStamp + 1) inThread:cThread messageBody:outgoingMessage.body attachments:outgoingMessage.attachments];
+        TSIncomingMessage *incomingMessage = [[TSIncomingMessage alloc] initWithTimestamp:(outgoingMessage.timestamp + 1) inThread:cThread messageBody:outgoingMessage.body attachments:outgoingMessage.attachments];
```

Comment:
ok that makes sense.

On Sun, Jan 25, 2015 at 2:49 PM, Frederic Jacobs <notifications@github.com>
wrote:

> In Signal/src/textsecure/Messages/TSMessagesManager.m
> <https://github.com/WhisperSystems/Signal-iOS/pull/508#discussion_r23509527>
> :
>
> > @@ -233,7 +233,7 @@ -(void)handleSendToMyself:(TSOutgoingMessage*)outgoingMessage {
> >      [self.dbConnection readWriteWithBlock:^(YapDatabaseReadWriteTransaction *transaction) {
> >          TSContactThread *cThread = [TSContactThread getOrCreateThreadWithContactId:[SignalKeyingStorage.localNumber toE164] transaction:transaction];
> >          [cThread saveWithTransaction:transaction];
> > -        TSIncomingMessage *incomingMessage = [[TSIncomingMessage alloc] initWithTimestamp:(outgoingMessage.timeStamp + 1) inThread:cThread messageBody:outgoingMessage.body attachments:outgoingMessage.attachments];
> > +        TSIncomingMessage *incomingMessage = [[TSIncomingMessage alloc] initWithTimestamp:(outgoingMessage.timestamp + 1) inThread:cThread messageBody:outgoingMessage.body attachments:outgoingMessage.attachments];
>
> Timestamp is a word, not the camel case of two words, so I thought we
> would stick to that.
> http://en.wikipedia.org/wiki/Timestamp
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/WhisperSystems/Signal-iOS/pull/508/files#r23509527>.
>



-- 
Dr. Corbett Moran
Rockets @ http://www.spacex.com
Physics @ http://www.ICS.uzh.ch  <http://www.ICS.uzh.ch>
Dev @ http://circleof6app.com
Dev @ https://whispersystems.org
www.christinecorbettmoran.com